packages/@expo/fingerprint/e2e/__tests__/updates-test.ts

```diff
@@ -0,0 +1,73 @@
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs/promises';
+import path from 'path';
+import rimraf from 'rimraf';
+
+import { createProjectHashAsync } from '../../src/Fingerprint';
+
+jest.mock('../../src/sourcer/ExpoConfigLoader', () => ({
+  // Mock the getExpoConfigLoaderPath to use the built version rather than the typescript version from src
+  getExpoConfigLoaderPath: jest.fn(() =>
+    jest
+      .requireActual('path')
+      .resolve(__dirname, '..', '..', 'build', 'sourcer', 'ExpoConfigLoader.js')
+  ),
+}));
+
+describe('updates support', () => {
```

Comment:
```suggestion
describe('updates managed support', () => {
```

(we should also add another test for updates generic support, i.e. one with checked in ios and android dirs and no `ignorePaths` supplied to `createProjectHashAsync` calls)

Reply:
having a question for testing generic workflow. the pseudo code is like this for generic workflow
```
create blank project
npx expo prebuild
const hash1 = await createProjectHashAsync()
const hash2 = await createProjectHashAsync()
expect(hash1).toBe(hash2);
```

the test looks not useful and confusing since no other operations between the two createProjectHashAsync(). does that make sense?

Reply:
Yep, agree that it's not useful if that's the equivalent here.